container_workflow_tool/git_operations.py

```diff
@@ -113,20 +113,24 @@ def _clone_upstream(self, url, ups_path, commands=None):
         os.chdir(oldcwd)
         return repo
 
-    def are_unpushed_commits_available(self, repo) -> bool:
+    def are_unpushed_commits_available(self, repo, branch_name="") -> bool:
         """
         Get unpushed commits
         :param repo: repo object to check for unpushed commits
+        :param branch_name: In case of gitlab, forked_branch and branch_name has to be defined.
+                            branch_name is e.g. rhel-8.7.0 and 'repo.active_branch.name' is 'rhel-8.7.0-<ubi_name>'
         :return: List of commits or empty array
         """
         branch = repo.active_branch.name
         # Get a list of commits that have not been pushed to remote
         select = "origin/" + branch + ".." + branch
+        if branch_name != "":
+            select = "origin/" + branch_name + ".." + branch
         if len(list(repo.iter_commits(select))) == 0:
```

Comment:
You can shorten this condition into:

```suggestion
        return bool(list(repo.iter_commits(select)))
```

It returns False for empty list and True otherwise.

But docstring says that this function should return a list of commits or emty array, which is no longer true.